client/components/Navlist.jsx

```diff
@@ -0,0 +1,31 @@
+/* eslint react/prop-types: 0 */
```

Comment:
Let's change this in `eslint.cjs`

Reply:
i couldn't get it to work. I tried a few different things. Instead I just added this in Navlist.jsx 
![image](https://user-images.githubusercontent.com/35232697/107263519-74185880-6a07-11eb-9aa2-8e113b186674.png)

Reply:
You can change add a key to rules in eslint for `"react/prop-types": 0`